services/src/main/java/edu/unc/lib/dl/cdr/services/processing/XMLExportJob.java

```diff
@@ -203,7 +199,7 @@ private File zipit(File mdExportFile) throws IOException {
         return mdExportZip;
     }
 
-    private void sendEmail(File mdExportFile, String toAddress) throws MessagingException {
+    private void sendEmail(File mdExportFile, String toAddress) {
```

Comment:
Is there still an exception on error, now that MessagingException is gone?

Reply:
No, the exception it was supposed to catch is actually swallowed here:
https://github.com/UNC-Libraries/Carolina-Digital-Repository/blob/69bcc5e865afc8e8c325e2b5ac3e81bc15e39505/persistence/src/main/java/edu/unc/lib/persist/services/EmailHandler.java#L62